easypost/__init__.py

```diff
@@ -621,19 +621,15 @@ def delete(self, **params):
 class Address(AllResource, CreateResource):
 
     @classmethod
-    def create(cls, api_key=None, verify=None, verify_strict=None, **params):
+    def create(cls, api_key=None, verify=[], verify_strict=[], **params):
```

Comment:
You can't use mutable types as kwargs.

Reply:
I suppose this could be called repeatedly. Uhg I'm not a fan of that bit of Python behavior